atomicapp/constants.py

```diff
@@ -52,10 +53,10 @@
 DEFAULT_PROVIDER = "kubernetes"
 DEFAULT_CONTAINER_NAME = "atomic"
 DEFAULT_NAMESPACE = "default"
-DEFAULT_ANSWERS = {
+DEFAULT_ANSWERS_FORMAT = {
     "general": {
-        "provider": DEFAULT_PROVIDER,
-        "namespace": DEFAULT_NAMESPACE
+        "provider": None,
+        "namespace": "default"
```

Comment:
I think we should leave this as it was. I know the change here was to specify "None" but I don't really think we need that and will comment on how/why.